02-FlexBox/assignment/css/components/_buttons.scss

```diff
@@ -0,0 +1,30 @@
+.btn{
+    display: inline-block;
+    padding: .75em 2.5em;
```

Comment:
I'd really consider using REM's instead of EM's. REM's are a lot easier to keep track of
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units